src/EditorFeatures/CSharpTest/Completion/CompletionProviders/ExplicitInterfaceMemberCompletionProviderTests.cs

```diff
@@ -79,8 +85,8 @@ class Bar : IGoo
      void IGoo.$$
 }";
 
-            await VerifyItemExistsAsync(markup, "Goo()");
-            await VerifyItemExistsAsync(markup, "Goo(int x)");
+            await VerifyItemExistsAsync(markup, "Goo", displayTextSuffix: "()");
+            await VerifyItemExistsAsync(markup, "Goo", displayTextSuffix: "(int x)");
```

Comment:
Should we also amend the completion provider tests to assert that the displayText of any item _anywhere_ never has a ()? Because otherwise we could make a similar mistake in the future?

Reply:
See above.  trying to timebox.  I'm not trying to eliminate the issue entirely.  I'm trying to make simple and easy and targetted fixes for the issue coming up here.